analysis_options.yaml

```diff
@@ -1,6 +1,7 @@
 include: package:pedantic/analysis_options.yaml
 
 analyzer:
+  exclude: ['lib/src/worker_protocol.pb.dart']
```

Comment:
is this working?

Reply:
Nope, was trying it and forgot to remove.